lib/js/atoms/Badge/Badge.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+	<div class="badge" :class="{ '-minor': color === BADGE_COLORS.MINOR }">
+		<div class="badge__icon">
+			<slot />
+		</div>
+		<div class="badge__label">{{ label }}</div>
+	</div>
+</template>
+
+<style lang="scss" scoped>
+@import '../../../styles/settings/colors';
+@import '../../../styles/settings/radiuses';
+@import '../../../styles/settings/typography';
+@import '../../../styles/settings/spacings';
+@import '../../../styles/atoms/icons';
+
+.badge {
+	border-radius: $radius-s;
+	display: flex;
+	padding: $space-xxxxs $space-xxs;
+
+	&.-minor {
+		background-color: $color-minor-background;
+	}
+
+	&__icon {
+		display: flex;
+		align-items: center;
+		margin-right: $space-xxs;
+		height: $icon-xxs;
+		width: $icon-xxs;
+
+		& > img {
+			height: $icon-xxs;
+			width: $icon-xxs;
+		}
+	}
```

Comment:
This will break as soon as we introduce `BadgeWithIcon`. IMO this should be a slot without any icon-related styling. I'd name it `badge_media` and leave only the positioning. Media styles should come from above, in the slot.